cubedash/summary/_stores.py

```diff
@@ -510,7 +511,7 @@ def search_items(
             query = query.where(DATASET_SPATIAL.c.id.in_(dataset_ids))
 
         if require_geometry:
-            query = query.where(DATASET_SPATIAL.c.footprint != None)
+            query = query.where(DATASET_SPATIAL.c.footprint is not None)
 
```

Comment:
`is not None` is usually good practice in normal Python, but not in sqlalchemy queries. sqlalchemy overrides the `!=` operator but can't override the "is not" operator, so this changes the query behaviour.

(I'm curious if the tests caught this -- the build's broken for an unrelated reason)

One alternative, if you're getting this from a lint error, is to switch to sqlalchemy's [`isnot()` function](https://docs.sqlalchemy.org/en/13/core/sqlelement.html#sqlalchemy.sql.expression.ColumnElement.isnot), but in the past I just suppressed that particular lint error in our DB modules because I find the operator more readable (and it's used more often in sqlalchemy examples).

Reply:
Ah, good catch. I changed all these to `is not` and then realised it broke all the queries. I didn't notice I hadn't put this one back.